src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,102 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { getPeopleWithParents } from '../helpers';
+import { Person } from '../types';
+import { useSearchParams } from 'react-router-dom';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
+import { getSortedPeople } from '../utils/getSortedPeople';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [hasLoadingError, setHasLoadingError] = useState<boolean>(false);
+  const [searchParams] = useSearchParams();
+
+  const query = searchParams.get('query');
+  const centuries = searchParams.getAll('centuries');
+  const sex = searchParams.get('sex');
+  const sortBy = searchParams.get('sort');
+  const order = searchParams.get('order');
+  //   // eslint-disable-next-line no-console
```

Comment:
If there is no need in that code - better to clean it up. If you plan on using this in future, please, drop a comment at the code with a short explanation why is it here.